sound/hda/ext/hdac_ext_bus.c

```diff
@@ -172,7 +172,6 @@ EXPORT_SYMBOL_GPL(snd_hdac_ext_bus_device_init);
 void snd_hdac_ext_bus_device_exit(struct hdac_device *hdev)
 {
 	snd_hdac_device_exit(hdev);
-	kfree(hdev);
 }
```

Comment:
hdev may be allocated by kzalloc(), please refer: sound/soc/sof/intel/hda-codec.c

Reply:
We should keep it consistent. I.e. we should use the same function to allocate hdec on both skylake and sof driver.